source/fundamentals/run-command.txt

```diff
@@ -55,17 +55,58 @@ the current member's role in the replica set, on a database:
 For a full list of database commands and corresponding parameters, see
 the :ref:`Additional Information section <addl-info-runcommand>`.
 
+.. _node-command-options:
+
+Command Options
+---------------
+
+You can specify optional command behavior for the ``command()``
+and ``runCursorCommand()`` methods.
+
+The ``command()`` method accepts a ``RunCommandOptions`` object. To learn
+more about the ``RunCommandOptions`` type, see the `API documentation <{+api+}/types/RunCommandOptions.html>`__.
+
+The ``runCursorCommand() method`` accepts a ``RunCursorCommandOptions``
+object. To learn more about the ``RunCursorCommandOptions`` type, see
+the `API documentation <{+api+}/types/RunCursorCommandOptions.html>`__.
+
+Starting in version 6.0 of the {+driver-short+}, you can pass only the
+following options to the ``command()`` method:
+
+- ``comment``
+- ``enableUtf8Validation``
+- ``raw``
+- ``readPreference``
+- ``session``
+
+You can set additional options, including a write or read
+concern, in the command document that you pass to
+the ``command()`` method. To learn more about the available options for
+a command, select it from the list of commands on the :manual:`Database
+Commands </reference/command/>` page of the Server manual.
+
+See the Server manual :manual:`entry
+</reference/command/>` for a command to learn about its associated
+options.
```

Comment:
Question:
Did you mean to include this? If so, it repeats same the in the previous paragraph (style guide issue and confusing user experience) and should be removed.